tromega/lib/views/home_view.dart

```diff
@@ -16,175 +19,203 @@ class HomeView extends StatefulWidget {
-                      child: RichText(
-                          text: TextSpan(
-                              text: 'Anderen Tag auswählen',
-                              style: const TextStyle(
-                                  color: Colors.black,
-                                  fontSize: 12,
-                                  fontWeight: FontWeight.w600,
+      body: fetching
+          ? const Center(child: CircularProgressIndicator())
+          : Container(
+              alignment: Alignment.center,
+              child: SizedBox(
+                width: 320,
+                height: 190,
+                child: Container(
+                  alignment: Alignment.center,
+                  decoration: BoxDecoration(
+                      color: Colors.white,
+                      boxShadow: [
+                        BoxShadow(
+                          color: Colors.grey.withOpacity(0.2),
+                          spreadRadius: 5,
+                          blurRadius: 10,
+                          offset:
+                              const Offset(0, 3), // changes position of shadow
+                        ),
+                      ],
+                      borderRadius:
+                          const BorderRadius.all(Radius.circular(10))),
+                  child: Container(
+                      alignment: Alignment.topCenter,
+                      child: Column(
+                        children: [
+                          const Padding(
+                            padding: EdgeInsets.fromLTRB(1.0, 15.0, 1.0, 1.0),
+                            child: Text(
+                              'Nächstes Training',
+                              style: TextStyle(
+                                  fontSize: 30,
+                                  fontWeight: FontWeight.bold,
                                   fontFamily: 'Raleway'),
-                              recognizer: TapGestureRecognizer()
-                                ..onTap = () {
-                                  Navigator.push(
-                                      context,
-                                      MaterialPageRoute(
-                                          builder: (context) =>
-                                              PlanOverview()));
-                                })),
-                    )
-                  ],
-                )),
-          ),
-        ),
-      ),
+                              textAlign: TextAlign.center,
+                            ),
+                          ),
+                          const Padding(
+                            padding: EdgeInsets.fromLTRB(1.0, 1.0, 1.0, 30.0),
+                            child: Text(
+                              'Legs',
+                              style: TextStyle(
+                                  fontSize: 20,
+                                  fontWeight: FontWeight.w400,
+                                  fontFamily: 'Raleway'),
```

Comment:
schau mal ob du in dem theme text mit diesem oder ähnlichen Style findest und nutze das dann mit `Theme.of(context).textTheme.<jeweiligeSchrift>`
auch bitte, alle files mal danach prüfen, sieht dann einheitlicher aus und spart zeilen